src/Pages/Nodes.razor

```diff
@@ -319,36 +385,71 @@
                     _nodes = await NodeRepository.GetAll();
 
                 }
-
+                await StopJob(node);
```

Comment:
I think you should delete the jobs, not just unschedule them. @Jossec101?

Reply:
The last time we we discussed about unschedule?

Reply:
we talked about this:
New node:
- Create Job
Deleting the node:
- interrupt and delete job
Disable node:
- interrupt and unschedule the job
Enable node:
- re-schedule job